pkg/scheduler/framework/plugins/legacy_registry.go

```diff
@@ -428,31 +435,17 @@ func NewLegacyRegistry() *LegacyRegistry {
 			}
 			return
 		})
+	registry.registerPriorityConfigProducer(EvenPodsSpreadPriority,
+		func(args ConfigProducerArgs) (plugins config.Plugins, pluginConfig []config.PluginConfig) {
+			plugins.PreScore = appendToPluginSet(plugins.PreScore, podtopologyspread.Name, nil)
+			plugins.Score = appendToPluginSet(plugins.Score, podtopologyspread.Name, &args.Weight)
+			return
+		})
 
-	// The following two features are the last ones to be supported as predicate/priority.
+	// The following features are the last ones to be supported as predicate/priority.
```

Comment:
Since the only feature left here is `ResourceLimitsPriority` should we make this comment specific to that? ie something like `ResourceLimits is the last feature to be supported as predicate/priority. TODO: Remove this check once it graduates to GA`

Reply:
done. Thanks.